### DISCUSSION TITLE
Persist cookies accross app restarts

### DISCUSSION THREAD

**Discussion by cuzox** [buildnatively/community#66] [Ideas]
### Use case

```Text
When the app is closed manually from the recent apps screen, the cookies are not persisted and the user needs to log in again. I'd like the cookies to not go away when the app is closed
```

### Idea

```Text
Not sure how best to make this happen
```

**Comment by galinalf** [comment]
would like to use cookies to store user login if they don't want to use biometrics

**Comment by romanfurman6** [comment]
Dear @cuzox and @galinalf,

I apologize for the delay in responding to your message. Our team was taking some time off for the 2024 holiday season.

It seems that your post may have been submitted to the wrong category. Please double-check the correct category by clicking on https://github.com/orgs/buildnatively/discussions/categories/q-a.

Additionally, please ensure that you are saving your cookies correctly by following the instructions at https://www.w3schools.com/html/html5_webstorage.asp.

If you require any further assistance, please do not hesitate to let me know.

**Comment by romanfurman6** [comment]
@galinalf @cuzox  did you guys solved it ?

**Comment by romanfurman6** [comment]
@galinalf @cuzox I think i was able to reproduce this issue. Pushed a fix!

**Reply by cuzox** [reply]
just checked it out, it works! thank you 🙌🏽
